installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json

```diff
@@ -93,5 +93,17 @@
 			 "logs": false,
 			 "assets": false
 		 }
-   }
+   },
+    "APIGEE": {
+        "endpoints": {
+            "prod": "{APIGEE_PROD_API}",
+            "test": "{APIGEE_TEST_API}",
+            "stg": "{APIGEE_STG_API}",
+            "dev01": "{APIGEE_DEV01_API}",
```

Comment:
We'll only have PROD, STG & DEV. Others will not be used by Jazz.

Reply:
why not test, we can use it for testing(unit/integration)